subscriptions/subscriptions.go

```diff
@@ -461,6 +490,12 @@ type SubscriptionResponse struct {
 	Message      string       `json:"message"`
 }
 
+type SubscriptionCancelResponse struct {
+	Subscription Subscription `json:"subscription"`
+	Code         int64        `json:"code"`
+	Message      string       `json:"message"`
+}
```

Comment:
The only thing I question here is whether the `Subscription` struct will work. In the docs the `"taxes"` key is included in the response, but the `Subscription` struct doesn't include that field. Like you saw with the `AddCharge...` method, Zoho returns an undocumented mutation to the `Subscription` data type.

Reply:
@bondar-pavel any thoughts so far? is your code working as is?

Reply:
Hi @schmorrison, sorry lost track of this PR for some time.
Cancel Subscription worked out for me, but from reply prospective I was mostly interested in SubscriptionID field, but not "taxes".
So I assume taxes field could be in reply and I just missed it.

Looks like there are a few options here:
1. Add Taxes under existent Subscription struct with omitempty tag
2. Create separate structure for this reply
3. Leave as is

But it becomes much harder to retest for me, as I no longer have access to zoho Subscription service (the middle of changing job right now). So testing the change become much harder for me, as requires to involve other folks with zoho access to retest.

Reply:
Lright. Ill merge as is, if someone notes the problem in the future we can deal with it then.